cluster-autoscaler/cloudprovider/gce/reserved.go

```diff
@@ -16,275 +16,11 @@ limitations under the License.
 
 package gce
 
-import (
-	"fmt"
-	"math"
-	"strconv"
-	"strings"
+// Reserved calculates the OS reserved values.
+type Reserved interface {
```

Comment:
I'd prefer a more descriptive name. Maybe 'NodeReservedCalculator'? Or 'ReservedResoucesCalculator'?

Reply:
Renamed to OsReservedCalculator